### PR TITLE
[INLONG-8794][Manager] Support add streamField and sinkField

### LINKED ISSUE #8794: [Improve][Manager] Support add streamField and sinkField
### Description

Support add streamField and sinkField and remove duplicates when adding fields.

### InLong Component

InLong Manager

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by fuweng11**
### Prepare a Pull Request

- Fixes #8794 

### Motivation

Support add streamField and sinkField and remove duplicates when adding fields.

### Modifications

Support add streamField and sinkField and remove duplicates when adding fields.

### CODE REVIEW

File: inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/sink/StreamSinkServiceImpl.java [hunk 1]
```diff
@@ -700,6 +703,39 @@ public Boolean updateAfterApprove(List<SinkApproveDTO> approveList, String opera
...
     }
 
+    @Override
+    public boolean addFields(StreamSinkEntity sinkEntity, List<SinkField> sinkFieldList) {
+        Set<String> existFields = sinkFieldMapper.selectBySinkId(sinkEntity.getId()).stream()
+                .map(StreamSinkFieldEntity::getFieldName).collect(Collectors.toSet());
+
+        LOGGER.debug("begin to save sink fields={}", sinkFieldList);
+        if (CollectionUtils.isEmpty(sinkFieldList)) {
+            return true;
```
Comment (vernedeng) [inline inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/sink/StreamSinkServiceImpl.java:713]: Please add more debug log

File: inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/sink/StreamSinkServiceImpl.java [hunk 2]
```diff
@@ -700,6 +703,39 @@ public Boolean updateAfterApprove(List<SinkApproveDTO> approveList, String opera
...
+            }
+            fieldEntity.setInlongGroupId(sinkEntity.getInlongGroupId());
+            fieldEntity.setInlongStreamId(sinkEntity.getInlongStreamId());
+            fieldEntity.setSinkType(sinkEntity.getSinkType());
+            fieldEntity.setSinkId(sinkEntity.getId());
+            fieldEntity.setIsDeleted(InlongConstants.UN_DELETED);
+            needAddFieldList.add(fieldEntity);
+        }
+        if (CollectionUtils.isNotEmpty(needAddFieldList)) {
+            sinkFieldMapper.insertAll(needAddFieldList);
```
Comment (vernedeng) [inline inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/sink/StreamSinkServiceImpl.java:733]: ditto

**Review by yunqingmoswu — COMMENTED** [review]
Maybe it is better to add some unit tests for it.

File: inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/sink/StreamSinkServiceImpl.java (hunk 1 above)
Comment (fuweng11) [reply inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/sink/StreamSinkServiceImpl.java:713]: The log above shows whether the field is empty.

**Comment by fuweng11** [comment]
> Maybe it is better to add some unit tests for it.

Done.

**Review by yunqingmoswu — APPROVED** [review]

**Review by vernedeng — APPROVED** [review]

**Review by dockerzhang — APPROVED** [review]

**Pull request merged**